### ISSUE TITLE
Programmatic mapping

### ISSUE THREAD

**Issue by yrodiere** [poblish/spring-boot-elasticsearch-sync#1]
I saw this comment:

https://github.com/poblish/spring-boot-elasticsearch-sync/blob/7afdc1ebc6a2583706f27b751e138c2a82fa3a13/src/main/kotlin/com/example/demo/model/Person.kt#L10

FYI, if you don't want Hibernate Search annotation in your entities, you can use programmatic mapping instead: https://docs.jboss.org/hibernate/search/6.0/reference/en-US/html_single/#mapper-orm-programmatic-mapping

**Comment by poblish** [comment]
Yes indeed, thanks very much for reminding me about that. If / when I get some time to extend this, I'll definitely have a go with that.
